urdf2model.py

```diff
@@ -1,63 +1,89 @@
 import os
```

Comment:
rename the file maybe too ( urdf 2 sdf ) And check other scripts in the repo, please. Examples seem to be using old methods.

Reply:
In the example file, I've renamed the method.